osquery/tables/networking/darwin/wifi.mm

```diff
@@ -0,0 +1,163 @@
+/*
+ *  Copyright (c) 2014, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <CoreFoundation/CoreFoundation.h>
+#include <Foundation/Foundation.h>
+#include <iomanip>
+
+#include <osquery/core.h>
+#include <osquery/tables.h>
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/sql.h>
+
+#include "osquery/core/conversions.h"
+
+#define kAirportPreferencesPath               \
+  "/Library/Preferences/SystemConfiguration/" \
+  "com.apple.airport.preferences.plist"
```

Comment:
As far as I can tell, this is the only location (at least on 10.9 and 10.10) where, if the option to remember known networks is enabled, they get persisted to disk here. If the known/remembered networks key doesn't exist, we just don't bother with it.

I did look at `SCDynamicStore`, but couldn't find any related keys there. And Airport/802.11 are private frameworks.